simulations/vip-260/bsctestnet.ts

```diff
@@ -0,0 +1,64 @@
+import { TransactionResponse } from "@ethersproject/providers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import { expectEvents } from "../../src/utils";
+import { forking, pretendExecutingVip, testVip } from "../../src/vip-framework";
+import { COMPTROLLER, NEW_BORROW_CAP, OLD_BORROW_CAP, WBETH_VTOKEN, vip260 } from "../../vips/vip-260/bsctestnet";
+import { abi as DIAMOND_CONSOLIDATED_ABI } from "./abi/DiamondConsolidated.json";
+
+forking(37940238, () => {
+  const provider = ethers.provider;
+  let comptroller: ethers.Contract;
+  before(async () => {
+    comptroller = new ethers.Contract(COMPTROLLER, DIAMOND_CONSOLIDATED_ABI, provider);
+  });
+  describe("Pre-VIP behavior", () => {
+    it("Verify current borrow cap is 200", async () => {
+      const currentWBethBorrowCap = await comptroller.borrowCaps(WBETH_VTOKEN);
+      expect(currentWBethBorrowCap).equals(OLD_BORROW_CAP);
+    });
+  });
+
+  describe("Post-VIP behavior", () => {
+    it("Verify current borrow cap is 200", async () => {
+      const currentWBethBorrowCap = await comptroller.borrowCaps(WBETH_VTOKEN);
+      expect(currentWBethBorrowCap).equals(OLD_BORROW_CAP);
+    });
+    it("Verify current borrow cap is 200", async () => {
+      await pretendExecutingVip(vip260());
+      const currentWBethBorrowCap = await comptroller.borrowCaps(WBETH_VTOKEN);
+      expect(currentWBethBorrowCap).equals(NEW_BORROW_CAP);
+    });
+  });
+});
```

Comment:
Seems redundant?

Reply:
yeah it is I added it while debugging while FastTrack was failing

Reply:
I think we can remove it.